scripts/db_stop.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ "$(docker ps -a -q -f name=mysql_dev)" ]; then
-  docker stop mysql_dev
-  docker rm -f mysql_dev
+if [ "$(docker ps -a -q -f name=mysql_db)" ]; then
+  docker stop mysql_db
+  docker rm -f mysql_db
 fi
```

Comment:
please leave a blank line at the end of each file

Reply:
Added blank lines.